tools/RAiDER/models/era5t.py

```diff
@@ -14,4 +14,5 @@ def __init__(self):
         self._Name = 'ERA-5T'
 
         self._valid_range = (datetime.datetime(1950, 1, 1), "Present")  # Tuple of min/max years where data is available.
-        self._lag_time = datetime.timedelta(days=5)  # Availability lag time in days
+        # Availability lag time in days; actually about 12 but unstable on ECMWF side
+        self._lag_time = datetime.timedelta(days=1)
```

Comment:
So you are making it 24 hours? Can you check what the documention states for ERA5T? and reference it in the text?